lmfdb/belyi/main.py

```diff
@@ -682,6 +675,11 @@ def belyi_search(info, query):
             a = query["abc_list"][0]
             query["$or"] = [{"a_s": a}, {"b_s": a}, {"c_s": a}]
         query.pop("abc_list")
+    return query
+
+def common_parse(info, query):
+    parse_bracketed_posints(info, query, "abc_list", "a, b, c", maxlength=3)
+    query = query_convert_abc_list(query)
```

Comment:
The query is modified in place, so you don't need to return it and set it agian.